uvicorn/protocols/websockets/wsproto_impl.py

```diff
@@ -35,11 +35,20 @@ def __init__(self, config, server_state):
         self.queue = asyncio.Queue()
         self.handshake_complete = False
         self.close_sent = False
-
-        self.conn = wsproto.connection.WSConnection(
-            conn_type=wsproto.connection.SERVER,
-            extensions=[wsproto.extensions.PerMessageDeflate()],
-        )
+        
+        if hasattr(wsproto, "WSConnection"):
+            self.conn = wsproto.WSConnection(
+                connection_type=wsproto.connection.SERVER,
+                #extensions=[wsproto.extensions.PerMessageDeflate()],
```

Comment:
Not sure where the extensions should gol the new class does not have an argument/attribute for it.

Reply:
These are now passed in the `AcceptConnection` event.